definitions/contested/json/CaseEvent/CaseEvent-consentOrder-nonprod.json

```diff
@@ -0,0 +1,15 @@
+[
```

Comment:
couldn't we get away with putting everything in the prod file except the caseEventAuthorisation file? would give us less to clean up at the end

Reply:
Very good point, I'll do this now